packages/camera/camera_platform_interface/lib/src/platform_interface/camera_platform.dart

```diff
@@ -272,6 +272,30 @@ abstract class CameraPlatform extends PlatformInterface {
     throw UnimplementedError('setZoomLevel() is not implemented.');
   }
 
+  /// Gets a list of video stabilization modes that are supported for the selected camera.
+  ///
+  /// Will return the list of supported video stabilization modes
+  /// on Android (when using camera_android_camerax package) and
+  /// on iOS. Will return an empty list on all other platforms.
+  Future<Iterable<VideoStabilizationMode>> getVideoStabilizationSupportedModes(
+      int cameraId) async {
+    throw UnimplementedError(
+        'getVideoStabilizationSupportedModes() is not implemented.');
+  }
+
+  /// Sets the video stabilization mode for the selected camera.
+  ///
+  /// On Android (when using camera_android_camerax) and on iOS
```

Comment:
Same.

Reply:
For the previous comment, I simply removed the paragraph. On this one, however, I just removed:

```dart
  /// On Android (when using camera_android_camerax) and on iOS
  /// the supplied [mode] value should be a mode in the list returned
  /// by [getVideoStabilizationSupportedModes].
```

But now I am in doubt about this one:
```dart
  ///
  /// Throws a [CameraException] when a not supported video stabilization
  /// mode is supplied.
```
It is assuming the behaviour expected by the specific platforms, but it can't really guarantee that they will behave as expected. Should this comment stay or shall I remove it?

Reply:
> But now I am in doubt about this one:
> 
> ```dart
>   ///
>   /// Throws a [CameraException] when a not supported video stabilization
>   /// mode is supplied.
> ```
> 
> It is assuming the behaviour expected by the specific platforms, but it can't really guarantee that they will behave as expected. Should this comment stay or shall I remove it?

There is a difference between an abstract interface's comment describing what an implementation of a method is required to do, and the comment making assertions about what the current implementations do.

The simple test in most cases is to ask: is this comment intended to be prescriptive about what a conforming implementation must do? If so, it makes sense; if not, it probably doesn't.

For instance:
> Will return an empty list on all other platforms.

Would it be a bug if someone implemented stabilization on Windows? Clearly not, which means this comment doesn't belong here.

> Throws a [CameraException] when a not supported video stabilization mode is supplied.

Would it be a bug if one implementation had divergent handling of invalid usage as compared to other platforms? Yes, because having predicable results to programming errors is a feature, so this does belong.